slice-test/slice-mapper-test/src/test/groovy/com/cognifide/slice/mapper/annotation/MappingStrategyTest.groovy

```diff
@@ -0,0 +1,37 @@
+package com.cognifide.slice.mapper.annotation
+
+import com.cognifide.slice.test.setup.BaseSetup
+import org.junit.Assert
+
+/**
+ * @author Mariusz Kubiś Date: 15.04.15
+ */
+class MappingStrategyTest extends BaseSetup {
```

Comment:
Please add fields with set values and check how they are overridden by Slice depending on strategy